docs/nx-cloud/reference/nx-cloud-cli.md

```diff
@@ -80,14 +80,13 @@ You can configure your CI run by passing the following flags:
 
 ### --distribute-on
 
-Tells Nx Cloud how many agents to use (and what launch templates to use) to distribute tasks. E.g.,
+By default, `npx nx-cloud start-ci-run` is intended for use with [Nx Agents](/ci/features/distribute-task-execution) and expects `--distribute-on` to be configured. It will output a warning if this flag is not set. If you are running a distributed execution with a legacy setup without Nx Agents, you can pass `--distribute-on=manual` to disable this warning.
```

Comment:
Move this paragraph to be the second paragraph in this section.  The definition of the flag should be the first paragraph and then this caveat paragraph